collector/collector.go

```diff
@@ -1,4 +1,15 @@
-// Copyright 2017 Apcera Inc. All rights reserved.
+// Copyright 2015-2018 The NATS Authors
```

Comment:
Why is prometheus exporter going back to 2015?

Reply:
Bad copy and paste... thanks!